### PR TITLE
[april fools testmerge] finally....obamium...

### PR DESCRIPTION
**Pull request by ghost**
:cl: Qustinnus
add: We have found material constructed from a former president of the Sol sector....What?
/:cl:

adds obamium. one of the strongest materials on earth. finally

todo: make textures work on worn overlays

image

### CODE REVIEW

**Comment by mrhugo13** [comment]
great pr

**Comment by trollbreeder** [comment]
speedmerge now

**Comment by TiviPlus** [comment]
https://www.youtube.com/watch?v=Tf00CkzB3oQ

**Comment by GunHog** [comment]
This PR is clearly powered by a desire to sit on Obama's face.

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -685,3 +685,14 @@ new /datum/stack_recipe("paper frame door", /obj/structure/mineral_door/paperfra
 	merge_type = /obj/item/stack/sheet/meat
 	material_type = /datum/material/meat
 	material_modifier = 1 //None of that wussy stuff
+
+/obj/item/stack/sheet/obamium
+	name = "obamium sheet"
+	desc = "Finally...Obamium"
+	singular_name = "obamium sheet"
+	icon_state = "sheet-meat"
```
Comment (Iamgoofball) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: add a sprite for this

**Comment by 81Denton** [comment]
absolute kino

**Comment by Fikou** [comment]
make obamium reagent that turns anytihng it touches into obamium

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 1 above)
Comment (Qustinnus) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: it's a greyscale sheet its fine

**Comment by LilPenpusher** [comment]
They thought it wasn't possible. They said we couldn't do it. But there was a voice calling us from within the void...


**Yes, we can.**

**Comment by Cyberboss** [comment]
I love this community

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 1 above)
Comment (Iamgoofball) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: yeah but you could make it the rotating pyramid from the meme

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 2]
```diff
@@ -685,3 +685,14 @@ new /datum/stack_recipe("paper frame door", /obj/structure/mineral_door/paperfra
 	merge_type = /obj/item/stack/sheet/meat
 	material_type = /datum/material/meat
 	material_modifier = 1 //None of that wussy stuff
+
+/obj/item/stack/sheet/obamium
+	name = "obamium sheet"
+	desc = "Finally...Obamium"
```
Comment (ATH1909) [inline code/game/objects/items/stacks/sheets/sheet_types.dm]: add a space after this ellipsis.

also consider adding a period to this description.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 3]
```diff
@@ -2094,6 +2094,40 @@
 	A.material_flags = applied_material_flags
 	A.set_custom_materials(metal_dat)
 
+/datum/reagent/obamium
+	name = "Obamium"
+	data = list("material"=null)
+	description = "A brown liquid made from a previous president. Can turn anything into his liking"
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: "Can turn anything into his liking" is worded a bit awkwardly.

also, period.

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 4]
```diff
@@ -2094,6 +2094,40 @@
...
+	color = "#341c02"
+	taste_mult = 100 //god the flavor
+	taste_description = "healthcare and drone-strikes" //what a legend
```
Comment (ATH1909) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: should "drone-strikes" have a hyphen in it?

**Comment by ATH1909** [comment]
Yo, what if we unironically merged this PR? It'll lay dormant (outside of adminbus) for many weeks until some madlad of a botanist discovers the Obamium chem in a packet of strange seeds, and then some wacky shit will happen.

**Comment by ATH1909** [comment]
also 
"add: We have found material constructed from a former president of the Sol sector....What?"
needs an "a" after "found".

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 4 above)
Comment (Qustinnus) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: i dont know im not fluent in american

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 2 above)
Comment (Qustinnus) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]: why dont u use code suggestions for things like this :^(

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 4 above)
Comment (Iamgoofball) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: should be drone strikes, not drone-strikes

File: code/game/objects/items/stacks/sheets/sheet_types.dm (hunk 2 above)
Comment (trollbreeder) [reply code/game/objects/items/stacks/sheets/sheet_types.dm]:
```suggestion
	desc = "Finally... Obamium..."
```

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 3 above)
Comment (trollbreeder) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]:
```suggestion
	description = "A brown liquid made from a previous president. Can turn anything that it touches into his face."
```

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 4 above)
Comment (trollbreeder) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]:
```suggestion
	taste_description = "healthcare and drone strikes" //what a legend
```

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 5]
```diff
@@ -2094,6 +2094,40 @@
 	A.material_flags = applied_material_flags
 	A.set_custom_materials(metal_dat)
 
+/datum/reagent/obamium
+	name = "Obamium"
+	data = list("material"=null)
+	description = "A brown liquid made from a previous president. Can turn anything that it touches into his face."
+	color = "#341c02"
+	taste_mult = 100 //god the flavor
```
Comment (MrDoomBringer) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: my sides

**Comment by cacogen** [comment]
my favourite black president

**Comment by LordSaladin** [comment]
we have obtained obamium

**Pull request closed without merging**